pkg/querier/postprocess.go

```diff
@@ -950,14 +950,25 @@ func (q *querier) prepareFillZeroArgsWithStep(functions []qbtypes.Function, req
 	copy(updatedFunctions, functions)
 
 	for i, fn := range updatedFunctions {
-		if fn.Name == qbtypes.FunctionNameFillZero && len(fn.Args) == 0 {
```

Comment:
Following the above, let's keep this unmodified, if the args are absent completely, we fill them otherwise expect the incoming payload to contain data.